docs/product-user-guide/src/main/asciidoc/decision-tables-upload-proc.adoc

```diff
@@ -7,13 +7,13 @@ IMPORTANT: You should typically upload only one decision table spreadsheet, cont
 
 .Procedure
 . Go to *Menu* -> *Design* -> *Projects* and click the project name.
-. Click *Create New Asset* -> *Decision Table (Spreadsheet)*.
+. Click *Add Asset* -> *Decision Table (Spreadsheet)*.
 . Enter an informative *Decision Table* name and select the appropriate *Package*.
 . Select the file type (*xls* or *xlsx*), click the *Choose File* icon, and select the spreadsheet. Click *Ok* to upload.
-. In the decision table designer, click *Validate* in the upper-right toolbar to validate the table. If the table validation fails, open the XLS or XLSX file and address any syntax errors. For syntax help, see xref:decision-tables-defining-proc[].
+. In the decision tables designer, click *Validate* in the upper-right toolbar to validate the table. If the table validation fails, open the XLS or XLSX file and address any syntax errors. For syntax help, see xref:decision-tables-defining-proc[].
 
 .Converting a decision table spreadsheet to a guided decision table
 [NOTE]
 ====
-To convert an uploaded decision table spreadsheet to a guided decision table, click *Convert* in the upper-right toolbar in the decision table designer. Converting the spreadsheet to a guided decision table enables you to edit the table data directly in {CENTRAL}. For more information about guided decision tables, see {URL_GUIDED_DECISION_TABLES}[_{GUIDED_DECISION_TABLES}_].
+To convert an uploaded decision table spreadsheet to a guided decision table, click *Convert* in the upper-right toolbar in the decision tables designer. Converting the spreadsheet to a guided decision table enables you to edit the table data directly in {CENTRAL}. For more information about guided decision tables, see {URL_GUIDED_DECISION_TABLES}[_{GUIDED_DECISION_TABLES}_].
```

Comment:
Is this correct:
decision tables designer

Reply:
Designer names are usually plural. Note the names in the Project Explorer next time your in the product. Thus same thing with generic references like this (guided rules designer, guided rule templates designer, decision tables designer, guided decision tables designer, etc.)